Buildings/Controls/OBC/Utilities/PIDWithAutotuning/AutoTuner/AMIGO/PI.mo

```diff
@@ -30,11 +30,12 @@ block PI "Identify control gain and integral time of the PI controller"
     "Time constant signal for the integral term"
     annotation (Placement(transformation(extent={{100,-50},{140,-10}}),
         iconTransformation(extent={{100,-80},{140,-40}})));
-  Buildings.Controls.OBC.Utilities.PIDWithAutotuning.AutoTuner.AMIGO.PIGain gai
-    "Calculate the control gain"
+protected
+  BaseClasses.PIGain                                                        gai
+    "Block that calculates the control gain"
     annotation (Placement(transformation(extent={{-10,20},{10,40}})));
-  Buildings.Controls.OBC.Utilities.PIDWithAutotuning.AutoTuner.AMIGO.PIIntegralTime
-    intTim "Calculate the integral time"
+  BaseClasses.PIIntegralTime
```

Comment:
full path, one line.

Reply:
The full path is used and the format is fixed